SU2_CFD/src/solver_adjoint_elasticity.cpp

```diff
@@ -91,10 +91,6 @@ CDiscAdjFEASolver::CDiscAdjFEASolver(CGeometry *geometry, CConfig *config, CSolv
   bool compressible = (config->GetKind_Regime() == COMPRESSIBLE);
   bool incompressible = (config->GetKind_Regime() == INCOMPRESSIBLE);
 
-  int rank = MASTER_NODE;
```

Comment:
However in this file all the declarations are removed but rank == MASTER_NODE is used in lines 686, 708, 1154, 1162

Reply:
I made rank/size a member of most of the bigger classes and initialize it only in the constructor. Sometimes, for smaller classes, where it is just used one or two times, I still create a local variable.